ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11-periodics.yaml

```diff
@@ -6533,14 +6615,6 @@ periodics:
     job-release: "4.11"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-aws-ipi-p3-f28-long-duration-part3-3
-  reporter_config:
-    slack:
-      channel: '#forum-prow-hive-qe-ci'
-      job_states_to_report:
-      - failure
-      - error
-      - success
-      report_template: Job {{.Spec.Job}} failed. <{{.Status.URL}}|View logs>
```

Comment:
Those removal are unexpected

Reply:
I'm not sure what happened here, maybe make jobs lost its mind at some point since I never edit this file directly. re-synched my branch with current upstream/master and restored the copy in master to my branch. the only diff I see at this point in the 4.11 periodics file is the addition of my aro job.